rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -598,6 +598,19 @@ def _verify_and_get_source_and_target_deployments(
             )
         return source_deployment, target_deployment
 
+    @staticmethod
+    def _get_deployment(sm, deployment_id, tenant_name):
+        if tenant_name:
+            return sm.get(models.Deployment,
+                          None,
+                          filters={'id': deployment_id,
```

Comment:
Maybe make a filters variable which always has 'id', and then add 'tenant_name' to it if tenant_name, then you can just have one sm.get call?
That's pretty much a style thing though, not a big deal either way.